### PR TITLE
Implement better tracking

### PR DESCRIPTION
**Pull request by jcreamer898**
Add a `track` decorator to allow for easy tracking.
Track a bunch of UI things.

### CODE REVIEW

File: spec/unit/decorators/track.spec.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
+// TODO: had to comment out because of https://github.com/deepsweet/istanbul-instrumenter-loader/issues/15
```
Comment (edubkendo) [inline spec/unit/decorators/track.spec.js]: https://medium.com/@kentcdodds/please-don-t-commit-commented-out-code-53d0b5b26d5f :trollface:

Comment (jcreamer898) [inline spec/unit/decorators/track.spec.js]: lolz

Comment (edubkendo) [inline spec/unit/decorators/track.spec.js]: Does mocha not have a way to mark specs as pending?

Comment (jcreamer898) [inline spec/unit/decorators/track.spec.js]: There is a bug in the code coverage tool that I'm using.

**Comment by edubkendo** [comment]
haruhithumbsup

**Pull request merged**